docs/view-managers.md

```diff
@@ -28,6 +28,13 @@ Currently, React Native for Windows uses a global `PerspectiveTransform` to prov
 
 __Important__: The `IsPerspectiveEnabled` property is experimental and support for it may be removed in the future.
 
+### Note about wrapping Windows controls
+As described above, some native UI components such as `ScrollView` and `TextInput` have been built by wrapping native Windows controls from the UWP XAML controls library. As a result, their visual appearance, animations, user experience, accessibility, etc. will always match their native Windows counterpart. 
+
+UI components can also be built using JavaScript. In this case, the component's source code does not wrap a native control, but rather combines already constructed React Native Windows components to build itself. Components built in this way contain no native code.

```

Comment:
does not _necessarily_/_directly_ wrap a native control
something will eventually draw some pixels on screen, through a native ui control, so at some point someone wraps a native control (unless people are directly drawing with win2d)

Reply:
Ok! I took the paragraph on line 32 from the tooltip statement we landed on for the "Wrapped Windows Control" badge in gallery. Should I be scrapping that paragraph and replacing with ^^? Or leave it and then follow with the note you mentioned above?

Reply:
TextInput at least shouldn't match the windows visual appearance.  TextInput in react-native is a primitive concept, not a full UI component.  -- And as such it doesn't match the windows visual appearance, as it shouldn't have any border etc like the native control has.

Reply:
Yes, this is an important point.

@chiaramooney / @stmoy  maybe it would be worth adding a page with a table of which UI components in RNW just wrap the native control versus not?